docs/faq.jade

```diff
@@ -316,6 +316,29 @@ block content
     var Kitten = connection.model('Kitten', kittySchema);
     ```
 
+    **Q**. How can I change mongoose's default behavior of initializing an array 
+    path to an empty array so that I can require real data on document creation?
+
+    **A**. You can set the default of the array to a function that returns null.
+    ```javascript
+    const CollectionSchema = new Schema({
+      field1: { 
+        type: [String], 
+        default: function() { 
```

Comment:
I think the more correct way here is `default: void 0`?

Reply:
done :) Thanks Val!